tests/python/gpu/test_operator_gpu.py

```diff
@@ -396,7 +396,7 @@ def _assert_all_almost_equal(lhs_list, rhs_list, rtol, atol):
             assert_almost_equal(lhs.asnumpy(), rhs.asnumpy(), rtol=rtol, atol=atol)
     if dtype == 'float16':
         rtol = 1e-3
-        atol = 1e-3
+        atol = 1e-2
     else:
         rtol = 1e-5
         atol = 1e-6
```

Comment:
what about when its !float16?

Reply:
I'm testing 10k including non fp16, so I guess the other tolerances are fine